pyAudioAnalysis/audioFeatureExtraction.py

```diff
@@ -25,12 +25,12 @@ def stZCR(frame):
 
 def stEnergy(frame):
     """Computes signal energy of frame"""
-    return numpy.sum(frame ** 2) / numpy.float64(len(frame))
+    return numpy.sum(numpy.absolute(frame ** 2)) / numpy.float64(len(frame))
```

Comment:
Looks like unnecessary use of numpy-namespaced functions. I believe this is equivalent to

```python
return abs(frame**2) / len(frame)
```

... or as noted, given that frame is an array of float and not complex, the `abs` can be removed as well. If performance is a concern, array multiplication runs significantly faster:

```python
return 1.0 / len(frame) * frame**2
```